drools-wb-services/drools-wb-verifier/drools-wb-verifier-webworker/src/main/java/org/drools/workbench/services/verifier/plugin/backend/DecisionTableVerifierWebWorkerProvider.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.drools.workbench.services.verifier.plugin.backend;
+
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.charset.StandardCharsets;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.kie.workbench.common.services.verifier.service.VerifierWebWorkerProvider;
+
+@ApplicationScoped
+public class DecisionTableVerifierWebWorkerProvider
+        implements VerifierWebWorkerProvider {
+
+    @Override
+    public String getId() {
+        return "dtableVerifier";
+    }
+
+    @Override
+    public String getWebWorker(final String fileName) throws Exception {
+        return loadResource(fileName);
+    }
+
+    public String loadResource(final String name) throws Exception {
+        final InputStream in = DecisionTableVerifierWebWorkerProvider.class.getResourceAsStream(name);
```

Comment:
You probably want to close this ```InputStream``` after its read?

Use of "try with resources" would work well.

Reply:
@manstis see #942

Reply:
Is there a quick way to merge this into my PR?

Reply:
@Rikkola Not AFAIK (to be honest copy and paste may be easier; it's only 2 changes).

If @jomarko had submitted a PR to your fork it'd be simple ;-) Otherwise I'm sure @jomarko and I can _pretend_ the changes are in your PR and we'll merge #942 after yours is merged.

Reply:
Good enough.

Reply:
Usually I do it in this way (PR to author's fork), but this time I decided to do separate PR to not mix, moving code and fix some issues.